cshape/hackathon-voting
# 6 implement scss with npm

This pull request updates the app to use SCSS, according to the process outlined here:
https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-a-sass-stylesheet

1. Updated react-scripts to version 2.0.3 so we can use the SCSS solution outlined in the create-react-app readme
2. Installed node-sass with npm
3. Renamed css files with extension `.scss`, updated dependent paths, and tested with a simple variable
4. Updated the README.md with instructions for running the app

To test:
1. Pull the branch
2. Run `npm install`
3. Run `npm start`
4. The app should start and render with no issues
